sdk-api-src/content/d3d11/ns-d3d11-d3d11_input_element_desc.md

```diff
@@ -58,15 +58,15 @@ A description of a single element for the input-assembler stage.
 
 Type: <b><a href="/windows/desktop/WinProg/windows-data-types">LPCSTR</a></b>
 
-The HLSL semantic associated with this element in a shader input-signature.
+The HLSL semantic associated with this element in a shader input-signature. See <a href="/windows/win32/direct3dhlsl/dx-graphics-hlsl-semantics">HLSL Semantics</a> for more info.
```

Comment:
Non-whitespace.

Reply:
FYI ,markdown is preferred over HTML.

Reply:
Thanks for giving me a heads up, I'll use pure Markdown going forward 🙂